workflow-runtime/src/commonMain/kotlin/com/squareup/workflow1/RuntimeConfig.kt

```diff
@@ -20,20 +20,33 @@ public annotation class WorkflowExperimentalRuntime
 /**
  * A specification of the Workflow Runtime.
  */
-public sealed interface RuntimeConfig {
+public sealed class RuntimeConfig {
+  /**
+   * If state has not changed from an action cascade, do not re-render. This has been mostly
+   * proven out. However, be careful if you have any non-Workflow code you integrate with that
+   * depends on the Workflow tree re-rendering to pick up changes from its equivalent 'view model.'
+   * You should change some kind of Workflow state when updating that external code if you want
+   * Workflow to pick up the change and render again.
```

Comment:
```suggestion
   * If state has not changed from an action cascade (as determined via `equals()`),
   * do not re-render. For example, when this is true and `noAction()` is enqueued,
   * the current `render()` pass will short circuit and no rendering will be posted
   * through the `StateFlow` returned from `renderWorkflowIn()`.
   *
   * This has been mostly proven out. However, be careful if you have any non-Workflow 
   * code you integrate with that depends on the Workflow tree re-rendering to pick up
   * changes from its equivalent 'view model.' You should change some kind of Workflow
   * state when updating that external code if you want Workflow to pick up the change
   * and render again.
```

Reply:
Used in the updated location in the enum.